src/__tests__/utilities/__tests__/integration-test.utilities.test.js

```diff
@@ -0,0 +1,196 @@
+import { readFile, validateMessageEquality } from "../integration-test.utilities";
+import { FileReadError } from "../../../errors/errors";
+import expect from "expect";
+
+describe('integration-test-utilities.js', () => {
+  describe('readMessage', () => {
+    it('should read a file successfully if the file name and folder path is correct', () => {
+      // given
+      const fileName = 'RCMR_IN030000UK06';
+      const folders = [
+        'equality-test',
+        'large-ehr-with-external-attachments',
```

Comment:
Maybe best to use the folder named 'large-ehr' instead for consistency, like the other related tests?